src/test/java/Search.java

```diff
@@ -0,0 +1,59 @@
+import InternSearch.pages.LoginPageSearch;
+import InternSearch.pages.SearchPageSearch;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Search extends BaseTest {
+    @Test(enabled = true, priority = 0)// dataProvider = "songTitles", dataProviderClass = BaseTest.class)
+    public void searchSongTitle() {
+        //GIVEN
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        //WHEN
+        searching.searchBoxMany();
+
+        //THEN
+        searching.resultFromSearch();
+    }
+    @Test(enabled = true,priority=1)
+    public void isSongSearchwithSentiveCase() {
+        //GIVEN
+        LoginPageSearch Login = new LoginPageSearch(driver);
+        SearchPageSearch searching = new SearchPageSearch(driver);
+        Login.login();
+
+        //WHEN
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
 
Hint:

**How can I beautify code in IntelliJ:**

The formatting shortcuts in Intellij IDEA are :
For Windows : Ctrl + Alt + L.
For Ubuntu : Ctrl + Alt + Windows + L.
For Mac : ⌥ (Option) + ⌘ (Command) + L.

![image](https://user-images.githubusercontent.com/104761482/209774134-9e63bfed-d768-4455-a860-028305c85e17.png)

Reply:
Thanks for your tips. I highlighted all and reformatted.